pages/onboarding_cluster.ipynb

```diff
@@ -0,0 +1,156 @@
+    "#####  Base resources\n",
+    "\n",
+    "Please run:\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!opfcli create-project $NAMESPACE_NAME $OWNER_TEAM -d $OPTIONAL_PROJECT_DESCRIPTION"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "This command will create:\n",
+    "\n",
+    "- A namespace in `cluster-scope/base/core/namespaces/$NAMESPACE_NAME`\n",
+    "- A blank user group for the `$OWNER_TEAM` if it does not exist yet in the `cluster-scope/base/user.openshift.io/groups/$OWNER_TEAM`\n",
+    "- An RBAC component for the project admin role `RoleBinding` in `cluster-scope/components/namespace-admin-rolebinding/$OWNER_TEAM` and maps it to the newly added namespace.\n",
+    "\n",
+    "##### Enabling namespace deployment to Zero cluster\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!cd cluster-scope/overlays/prod/$ENV/$TARGET_CLUSTER"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Open the `kustomization.yaml` file in this folder and add `../../base/core/namespaces/$NAMESPACE` to the `resources` field. We aim to keep this field alphabetically sorted for better human readability:\n",
+    "```\n",
+    "---\n",
+    "apiVersion: kustomize.config.k8s.io/v1beta1\n",
+    "kind: Kustomization\n",
+    "\n",
+    "resources:\n",
+    "...\n",
+    "- ../../base/core/namespaces/$NAMESPACE\n",
+    "...\n",
+    "```\n",
+    "#### Populate your team\n",
+    "The following steps enable users to access designated cluster/namespaces. This consists of adding users to the appropriate OpenShift groups.\n",
+    "\n",
+    "Navigate to `cluster-scope/base/user.openshift.io/groups/$NAME/`.\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!cd cluster-scope/base/user.openshift.io/groups/$NAME/"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Edit group.yaml file by adding github username of every user in the group.\n",
+    "```\n",
+    "---\n",
+    "apiVersion: user.openshift.io/v1\n",
+    "kind: Group\n",
+    "metadata:\n",
+    "  name: $GROUP_NAME\n",
+    "users:\n",
+    "  - user_1\n",
+    "  - user_2\n",
+    "```\n",
+    "```{note}\n",
+    "Github username **must** be converted to lowercase!\n",
+    "```"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "#### Finalize\n",
+    "Please stage your changes and send them as a PR against the [operate-first/apps](https://github.com/operate-first/apps) repository. Make sure:\n",
```

Comment:
Show a list of modified files. Something that the user can compare against that his PR will include proper changes.

Reply:
Resolved in latest commit.